src/main/java/calculator/StringCalculator.java

```diff
@@ -0,0 +1,19 @@
+package calculator;
+
+import java.util.List;
+
+public class StringCalculator {
+    private static final int FIRST_OPERAND_INDEX = 0;
```

Comment:
클래스 정적 상수와 생성자는 줄바꿈으로 구분해주면 가독성이 좋아집니다.
보통 컨벤션으로 인스턴스 필드와도 줄바꿈으로 구분하는 방식으로 표현을 합니다.
공유드립니다. 😄
```suggestion
    private static final int FIRST_OPERAND_INDEX = 0;
    
```